types/stylelint/index.d.ts

```diff
@@ -110,7 +110,7 @@ declare namespace stylelint {
 		rules?: ConfigRules;
 		quiet?: boolean;
 		defaultSeverity?: Severity;
-		ignoreDisables?: DisableSettings;
```

Comment:
see https://github.com/stylelint/stylelint/commit/b47df94515ca8e1114216e2345ed7e12030561ab#diff-9a4b261e295bef1ccce813aa194ac4023a1dcf98236667889964b1ba6fbc551fR42

Reply:
I have reviewed the usage of `ignoreDisables` in the codebase and the documentation.
It's a `boolean`, i.e. not `DisableSettings`.

Iv just linked when it was changed, to give some context.